rust-packages/ic-verifiable-credentials/src/lib.rs

```diff
@@ -1360,145 +1421,156 @@ mod tests {
-            &test_ic_root_pk_raw(),
-            CURRENT_TIME_BEFORE_EXPIRY_NS,
+            &vp_vc_spec(),
+            &mainnet_ic_root_pk_raw(),
+            VP_CURRENT_TIME_BEFORE_EXPIRY_NS,
         );
         assert_matches!(result, Err(e) if format!("{:?}", e).to_string().contains("InconsistentCredentialJwtClaims"));
     }
 
     #[test]
     fn should_fail_validate_ii_presentation_and_claims_if_wrong_effective_subject() {
-        let id_alias = Principal::from_text(ID_ALIAS_FOR_VP).expect("wrong principal");
-        let id_dapp = Principal::from_text(ID_RP_FOR_VP).expect("wrong principal");
+        let id_alias = Principal::from_text(VP_ID_ALIAS).expect("wrong principal");
+        let id_dapp = Principal::from_text(VP_RP_ID).expect("wrong principal");
         let vp_jwt = build_ii_verifiable_presentation_jwt(
             id_dapp,
-            ID_ALIAS_VC_FOR_VP_JWS.to_string(),
-            VERIFIED_ADULT_VC_FOR_VP_JWS.to_string(),
+            VP_ID_ALIAS_JWS.to_string(),
+            VP_VC_JWS.to_string(),
         )
         .expect("vp-creation failed");
         let result = validate_ii_presentation_and_claims(
             &vp_jwt,
             id_alias, // wrong effective subject
-            &default_test_vc_flow_signers(),
-            &verified_adult_vc_spec(),
-            &test_ic_root_pk_raw(),
-            CURRENT_TIME_BEFORE_EXPIRY_NS,
+            RP_DERIVATION_ORIGIN.to_string(),
+            &mainnet_test_vc_flow_signers(),
+            &vp_vc_spec(),
+            &mainnet_ic_root_pk_raw(),
+            VP_CURRENT_TIME_BEFORE_EXPIRY_NS,
         );
         assert_matches!(result, Err(e) if format!("{:?}", e).to_string().contains("unexpected vc subject"));
     }
 
     #[test]
     fn should_fail_validate_ii_presentation_and_claims_if_expired() {
-        let id_dapp = Principal::from_text(ID_RP_FOR_VP).expect("wrong principal");
+        let id_dapp = Principal::from_text(VP_RP_ID).expect("wrong principal");
         let vp_jwt = build_ii_verifiable_presentation_jwt(
             id_dapp,
-            ID_ALIAS_VC_FOR_VP_JWS.to_string(),
-            VERIFIED_ADULT_VC_FOR_VP_JWS.to_string(),
+            VP_ID_ALIAS_JWS.to_string(),
+            VP_VC_JWS.to_string(),
         )
         .expect("vp-creation failed");
         let result = validate_ii_presentation_and_claims(
             &vp_jwt,
             id_dapp,
-            &default_test_vc_flow_signers(),
-            &verified_adult_vc_spec(),
-            &test_ic_root_pk_raw(),
-            CURRENT_TIME_AFTER_EXPIRY_NS,
+            RP_DERIVATION_ORIGIN.to_string(),
+            &mainnet_test_vc_flow_signers(),
+            &vp_vc_spec(),
+            &mainnet_ic_root_pk_raw(),
+            VP_CURRENT_TIME_AFTER_EXPIRY_NS,
         );
         assert_matches!(result, Err(e) if format!("{:?}", e).to_string().contains("credential expired"));
     }
 
     #[test]
-    fn should_fail_validate_ii_presentation_and_claims_if_wrong_vcs() {
```

Comment:
I changed this test to make it more obvious. The only difference before between `VERIFIED_EMPLOYEE_VC_FOR_VP_JWS` and `VERIFIED_ADULT_VC_FOR_VP_JWS` was the credential spec. 

Therefore, the test is the same as pass a wrong spec.